### PR TITLE
Add initial structure for example tests

### PR DESCRIPTION
**Pull request by aliscott**
This also adds an example for Terragrunt

### CODE REVIEW

**Review by alikhajeh1 — COMMENTED** [review]
This is great! In master's readme, I pushed a tweak so we link to the individual examples so users can find the one relevant to them quickly. Can you please also add a 1-line description of what the example in the example's README? The idea being we can copy/paste that into the repo readme and the examples/readme.

Also what did you and @tim775 decide for versioning of GHA in the examples? They should all point to @v1? Can we create a tag and keep updating that tag in sync with master during dev in the next few days?

File: examples/terragrunt/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    name: Terragrunt
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      
+      - name: Install Terraform
+        uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_wrapper: false # This is required so that Terraform binary outputs valid JSON
```
Comment (alikhajeh1) [inline examples/terragrunt/README.md:23]: Should we report this to them? is https://github.com/hashicorp/setup-terraform/issues/42 the issue?

Comment (aliscott) [reply examples/terragrunt/README.md:23]: Yeah, that's the same issue.

**Review by tim775 — APPROVED** [review]

**Comment by aliscott** [comment]
> Also what did you and @tim775 decide for versioning of GHA in the examples? They should all point to @v1?

Yeah I think we should use v1 for the examples. When you run the tests it should replace it all with the local path anyway so no need to update it for dev yet.

**Pull request merged**